integration-test/tests/email-autofill.macos.spec.js

```diff
@@ -406,9 +406,30 @@ test.describe('macos', () => {
                 .applyTo(page)
 
             const perfPage = scannerPerf(page)
-            perfPage.navigate(constants.pages['perf-huge-regex'])
+            await perfPage.navigate(constants.pages['perf-huge-regex'])
 
             await perfPage.validateInitialScanPerf(80)
         })
+
+        /**
+         * See: https://app.asana.com/0/0/1207840563867787/f
+         *
+         * This covered a form with about 60 inputs, but none are deemed 'fillable'.
+         * In that case, we want to ensure we don't do any repeated work.
+         *
+         */
+        test('Large form without eligible inputs', async ({page}) => {
+            test.setTimeout(5000)
```

Comment:
Why is this needed?

Reply:
If I remember correctly, this just fails early. It times out at 5 seconds instead of the usual 30 seconds.